src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -80,10 +80,44 @@ public void storeMessage(Message message) {
 
     return messages;
   }
+  
   /** Returns the total number of messages for all users. */
   public int getTotalMessageCount(){
     Query query = new Query("Message");
+    int entityLimit = 1000;
```

Comment:
I think `entityLimit` is a bit vague. Can we be more explicit? I just checked the [FetchLimit documentation](https://cloud.google.com/appengine/docs/standard/java/javadoc/com/google/appengine/api/datastore/FetchOptions) and it says that this `is the maximum number of results the query will return`. So, maybe something like `maxNumberOfResultsPerQuery` would be more helpful.

> 

>